page-optimize.php

```diff
@@ -21,6 +24,36 @@
 	exit;
 }
 
+function page_optimize_cache_cleanup( $age = DAY_IN_SECONDS ) {
```

Comment:
Decided to be less aggressive after your comment @brandonpayton. One empty folder doesn't mean much (just deleting files inside the defined cache dir), and we now match by pattern.